build.yaml

```diff
@@ -1204,3 +1203,55 @@ job_type:
                    *package_jenkins_osx_yosemite_box ]
           }
       timeout: 120
+
+#-----------------------------------------------------------------------------#
+# View definitions below this point
+#-----------------------------------------------------------------------------#
+
+views:
+  trial:
+    description: 'All trial jobs'
+    regex: 'run_(?i)trial_.*.*'
+  acceptance:
+    description: 'All Acceptance jobs'
+    regex: 'run_(?i)acceptance_.*.*'
+  client:
+    description: 'All client instalation jobs'
+    regex: 'run_(?i)client_.*.*'
+  code_quality:
+    description: 'All code quality jobs'
+    regex: 'run_(?i)lint.*.*'
+  docs:
+    description: 'Documentation jobs'
+    regex: 'run_(?i)sphinx.*.*'
+  cinder:
+    description: 'cinder jobs'
+    regex: '.*_(?i)cinder_.*.*'
+  ebs:
+    description: 'ebs jobs'
+    regex: '.*_(?i)ebs_.*.*'
+  storage_drivers:
+    description: 'All Storage Driver Jobs'
+    regex: '.*_(?i)storage_(?i)driver_.*.*'
+  AWS:
+    description: 'All AWS Jobs'
+    regex: '.*_(?i)aws_.*.*'
+  Rackspace:
+    description: 'All Rackspace Jobs'
+    regex: '.*_(?i)rackspace_.*.*'
+  CentOS:
+    description: 'All CentOS jobs'
+    regex: '.*_(?i)centos_.*.*'
+  Trusty:
+    description: 'All Ubuntu Trusty jobs'
+    regex: '.*_(?i)trusty_.*.*'
+  Vivid:
+    description: 'All Ubuntu Vivid jobs'
+    regex: '.*_(?i)vivid_.*.*'
+  Wily:
+    description: 'All Ubuntu Wily jobs'
+    regex: '.*_(?i)wily_.*.*'
+  cron:
+    description: 'All Nightly Cron jobs'
+    regex: '^_[^_](?i)[a-z]*.*'
```

Comment:
I don't really know what these actually are, but I think this would be a great place for a longer (possibly even multi-paragraph) description of what they are and how they relate to the rest of the build.